File: App/app/src/main/java/hr/kcosic/app/model/enums/ErrorCodeEnum.kt
package hr.kcosic.app.model.enums

enum class ErrorCodeEnum(val code: Int) {
    UnexpectedError(0),
    InvalidCredentials(1),
    InvalidToken(2),
    UsernameExists(3),
    EmailExists(4),
    InvalidParameter(5),
    LegalNameExists(6),
    ShortNameExists(7),
    VatExists(8),
    RecordNotFound(9),
    NoReview(10),
}

File: WebAPI/WebAPI/Models/Enums/ErrorCodeEnum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebAPI.Models.Enums
{
    public enum ErrorCodeEnum
    {
        UnexpectedError,
        Forbidden,
        InvalidToken,
        UsernameExists,
        EmailExists,
        InvalidParameter,
        LegalNameExists,
        ShortNameExists,
        VatExists
    }
}